designsafe/apps/api/notifications/tests.py

```diff
@@ -176,3 +176,58 @@ def test_webhook_interactive_post(self, get_user_model, mock_receiver):
         self.assertEqual(action_link, link_from_event)
         self.assertEqual(n.operation, 'web_link')
         self.assertEqual(response.status_code, 200)
+class NotificationsViewsTests(TestCase):
```

Comment:
Double space before a `class` please. Check your code with `flake8` and `pylint` if possible.